src/main/java/com/openclassroomsproject/paymybuddy/configuration/security/CustomRequestCache.java

```diff
@@ -0,0 +1,15 @@
+package com.openclassroomsproject.paymybuddy.configuration.security;
+
+import org.springframework.security.web.savedrequest.HttpSessionRequestCache;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+class CustomRequestCache extends HttpSessionRequestCache {
+
+	@Override
+	public void saveRequest(HttpServletRequest request, HttpServletResponse response) {
+		if (!SecurityUtils.isFrameworkInternalRequest(request)) {
+			super.saveRequest(request, response);
```

Comment:
Why inheritance instead of interface?

Reply:
I extend HttpSessionRequestCache which is a spring security class which implement RequestCache. I can implement RequestCache but that mean i have to redifine all the class method.
Here, i just want to save request client to be able to redirect after the login. 
Inheritance is convenient here,no  ?

Reply:
Okay sounds good to me! :)